### DISCUSSION TITLE
make -j4 not working

### DISCUSSION THREAD

**Discussion by Safqut** [MDIL-SNU/SevenNet#111] [Q&A] [answered]
I followed the instructions as written in the readme.md part. I was able to use build the cmake files using this command:
"cmake ../cmake -DCMAKE_PREFIX_PATH=E:/Clemson/SevenNetProject/venv/Lib/site-packages/torch/share/cmake -DMKL_INCLUDE_DIR=E:/Clemson/SevenNetProject/venv/Library/include"
but then when I used "make -j4" it showed error. I tried different way to like using this command "cmake --build . --config Release" and this gave the error in the attached screenshot that it was not able to build the pair_e3_gnn.cpp files!
![Screenshot (672)](https://github.com/user-attachments/assets/c9558627-b19e-43ee-a955-0d23e8cdd980)
Please help me in this case.

**Comment by YutackPark** [comment, marked as answer]
I found that one of error you're facing, C2131, is a windows specific error (but it make sense, sorry for bad code), but SevenNet is not tested on the windows. I know MINGW64 is for using gcc/g++ in windows but I think it still uses compiler for windows under the hood and this is the cause of an error. I'm not sure whether there exist a easy bypass for this case.

I think it would be nice to use SevenNet on windows, therefore, I'll mark this as an issue.

Here is some alternatives you may want:
1. If you just need MD simulator for SevenNet, there is a ASE calculator interface to run MD simulations or anything it support.
2. Wsl (windows subsystem for linux).

Note that other python parts (including training) are also not tested on windows and may cause unexpected errors.

Plus: You need `cuda_runtime.h` to build `pair_e3gnn_parallel`. But if you want only serial LAMMPS version (which is absolutely fine for non-HPC platform), you can safely remove `pair_e3gnn_parallel.h` and `pair_e3gnn_parallel.cpp` from `{path to LAMMPS}/src` and move on.

**Reply by Safqut** [reply]
Thank you for your suggestions. I swicthed to Linux and it worked.
